packages/app/src/components/profile/CardCollection.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement } from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+import { Card } from '../UI';
+
+type Props = {};
+
+export const CardCollection = (props: Props): ReactElement => {
```

Comment:
Should this be called the Card Collection component? I think these are just buttons that will open up to new pages